play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/config/TrackingConsentConfigSpec.scala

```diff
@@ -50,9 +50,21 @@ class TrackingConsentConfigSpec extends AnyWordSpec with Matchers {
       config.optimizelyUrl should equal(None)
     }
 
+    "return None if optimizely.projectId is null" in {
+      implicit val application: Application = buildApp(
+        Map(
+          "optimizely.url"       -> "http://optimizely.com/",
+          "optimizely.projectId" -> null
+        )
+      )
+      val config                            = application.injector.instanceOf[TrackingConsentConfig]
+      config.optimizelyUrl should equal(None)
+    }
+
     "return None if optimizely.url is not defined" in {
       implicit val application: Application = buildApp(
         Map(
+          "optimizely.url"       -> null,
```

Comment:
this value is set by default now in reference.conf so we have to zero it out here to check that it's required